### PR TITLE
gui seq diagrams

### PR DESCRIPTION
**Pull request by vercibar**

### CODE REVIEW

**Comment by triszt4n** [comment]
All done?

**Comment by vercibar** [comment]
as far as i know its done

File: uml/gui_seq_diagrams/draw.plantuml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+title "Game gets drawn"
+
+participant ": TurnController" as jvm
+participant "gameFrame: GameFrame" as gameFrame
+
+jvm -> gameFrame ++: UpdateComponent()
+
+gameFrame -> jvm --
+
+@enduml
```
Comment (triszt4n) [inline uml/gui_seq_diagrams/draw.plantuml:14]: This file isn't even needed, you've already made `update_gameframe.plantuml`.

**Review by triszt4n — APPROVED** [review]
I've removed unnecessary files in the commit above. One plantuml seq dia is unnecessary. Everything else is fine.

File: uml/gui_seq_diagrams/draw.plantuml (hunk 1 above)
Comment (triszt4n) [reply uml/gui_seq_diagrams/draw.plantuml:14]: Sorry, this review should've been a request for cahnge

File: uml/gui_seq_diagrams/mouseclicked_environmentpanel.plantuml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+end
+
+alt right mouse button and clicked on an asteroid or a teleportgate
+
+    participant "popup: SettlerActionPopup" as settlerpopup
+    participant "gameFrame: GameFrame" as gameFrame
+    participant "settler: Settler" as settler
+
+    env -> gameFrame ++: GetPlayer()
```
Comment (triszt4n) [inline uml/gui_seq_diagrams/mouseclicked_environmentpanel.plantuml:42]: hmm getcurrentplayer should come from turncontroller

Comment (triszt4n) [reply uml/gui_seq_diagrams/mouseclicked_environmentpanel.plantuml:42]: merge latest commits from ref/class-rethinking plesae

**Review by triszt4n — CHANGES_REQUESTED** [review]

File: uml/gui_seq_diagrams/mouseclicked_environmentpanel.plantuml (hunk 2 above)
Comment (triszt4n) [reply uml/gui_seq_diagrams/mouseclicked_environmentpanel.plantuml:42]: I'm getting more and more chaotic sossry

**Comment by triszt4n** [comment]
Na most magyarul: a turncontrollerből szerzik mostantól a diagramok a jelenlegi játékost. Kicsit kiegészítettem a draw game sequenciát, hogy ne sírjon a szájuk

**Review by triszt4n — APPROVED** [review]
# 🍭

**Pull request merged**